### PR TITLE
missing sendReport function

### PR DESCRIPTION
**Pull request by arunge**
### Bug Fixes
- add missing report function

### CODE REVIEW

File: R/01-etl.R [hunk 1]
```diff
@@ -53,8 +53,10 @@ report.default <- function(x, ...) {
   updated <- data.frame(
     tableName = x$name,
     completed = as.character(Sys.time()),
+    # hard-coded: currently mappingIds exist already on the DB but not yet on the beta branch
+    mappingId = "IsoMemo",
     nrow = nrow(x$dat),
```
Comment (arunge) [inline R/01-etl.R:58]: @jan-abel-inwt Could you check if the column names match to those on the server? Thanks!

File: R/01-etl.R [hunk 2]
```diff
@@ -53,8 +53,10 @@ report.default <- function(x, ...) {
   updated <- data.frame(
     tableName = x$name,
```
Comment (arunge) [inline R/01-etl.R:54]: @jan-abel-inwt Could you check if the column names match to those on the server? Thanks!

Comment (jan-abel-inwt) [reply R/01-etl.R:54]: image

File: R/01-etl.R [hunk 3]
```diff
@@ -53,8 +53,10 @@ report.default <- function(x, ...) {
...
     completed = as.character(Sys.time()),
```
Comment (jan-abel-inwt) [inline R/01-etl.R:55]:
```suggestion
    timestamp = as.character(Sys.time()),
```

File: R/01-etl.R (hunk 1 above)
Comment (jan-abel-inwt) [inline R/01-etl.R:58]:
```suggestion
    rows = nrow(x$dat),
```

File: R/01-etl.R (hunk 2 above)
Comment (jan-abel-inwt) [inline R/01-etl.R:54]:
```suggestion
    source = x$name,
```

**Review by jan-abel-inwt — CHANGES_REQUESTED** [review]
These are the correct column names from the `updated` table

**Review by jan-abel-inwt — APPROVED** [review]
looks fine to me :)

**Pull request merged**